src/pages/index.js

```diff
@@ -59,54 +32,126 @@ const config = {
   inputErrorClass: "modal__input_type_error",
   errorClass: "modal__error_visible",
 };
+const profileButton = document.querySelector(".profile__button");
+
+const profileAvatarModal = document.querySelector("#profile-avatar-modal");
+const profileImage = document.querySelector(".profile__image");
+const profileAvatarModalForm = profileAvatarModal.querySelector(".modal__form");
+
+const userInfo = new UserInfo(".profile__title", ".profile__description");
+const cardSection = new Section(renderCard, cardListEl);
+
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "f1dab6ed-e5ba-44ef-bef1-7369d7e9bf0d",
+    "Content-Type": "application/json",
+  },
+});
+
+api.getUserInfo().then((res) => {
+  userInfo.setUserInfo(res);
+});
+api.getInitialCards().then((res) => {
+  cardSection.renderItems(res);
+});
+api.getUserInfo().then((res) => {
+  profileImage.src = res.avatar;
+});
```

Comment:
`api.getUserInfo()` should be called only **once** to set all 3 params of the profile data